proto/cosmos/base/reflection/v2alpha1/reflection.proto

```diff
@@ -99,12 +99,6 @@ message InterfaceAcceptingMessageDescriptor {
   repeated string field_descriptor_names = 2;
 }
 
-// ConfigurationDescriptor contains metadata information on the sdk.Config
-message ConfigurationDescriptor {
-  // bech32_account_address_prefix is the account address prefix
-  string bech32_account_address_prefix = 1;
```

Comment:
@julienrbrt thoughts on putting this in auth instead of global server since we are dropping the globals